comictaggerlib/taggerwindow.py

```diff
@@ -1155,12 +1170,22 @@ def commit_metadata(self) -> None:
                 QtWidgets.QApplication.setOverrideCursor(QtGui.QCursor(QtCore.Qt.CursorShape.WaitCursor))
                 self.form_to_metadata()
 
-                success = self.comic_archive.write_metadata(self.metadata, self.save_data_style)
+                fail_list = []
+                # Save each style
+                for style in self.save_data_styles:
+                    success = self.comic_archive.write_metadata(self.metadata, style)
+                    if not success:
+                        fail_list.append(style)
```

Comment:
This should break, I'm not sure what happened you had fixed this earlier

Reply:
Me too. I must have lost it in the squish...